cmdb/file_export/export_types.py

```diff
@@ -185,21 +196,32 @@ def export(self, object_list, *args):
             Json file containing the object_list
         """
 
-        s_keys = ['public_id', 'active', 'type_id', 'fields']
-        filter_dict = []
+        output = []
 
         for obj in object_list:
-            new_dict = {'public_id': 0, 'active': True, 'type': '', 'fields': []}
-            for k in s_keys:
-                if k in obj.__dict__:
-                    if k == 'type_id':
-                        new_dict.update({'type': object_manager.get_type(obj.type_id).label})
-                    else:
-                        new_dict.update({k: obj.__dict__[k]})
+            # init output element
+            output_element = {}
+            output_element['public_id'] = obj.public_id
+            output_element['active'] = obj.active
+            output_element['type'] = type_manager.get(obj.type_id).get_label()
+            output_element['fields'] = []
+
+            # get object fields as dict:
```

Comment:
Just a little hint: You can make such conversions relatively easy using the collection methods _dict_ and _zip_. They would not bring any real advantage here, but they might be worth mentioning.